encord/utilities/benchmark_utilities.py

```diff
@@ -0,0 +1,210 @@
+from typing import Dict, List
+
+from shapely.geometry import Polygon, box
+
+from encord.client import EncordClientProject
+
+
+def get_project_labels_consensus(
+    baseline_project: EncordClientProject,
+    comparing_projects: List[EncordClientProject],
+    label_types: List[str],
+    threshold: float = 0.7,
+) -> Dict[str, Dict[str, float]]:
```

Comment:
From the return signature I am not 100% sure what can be expected here. If possible, could you maybe create a dataclass for the return type or clarify what is expected a bit better? If you give me some examples I could also suggest something.